client/Components/map.jsx

```diff
@@ -19,6 +25,30 @@ const Map = (props) => {
     clearEggData();
   };
 
+  useEffect(() => {
+    const getEggs = () => {
+      if (!user) return;
+      const req = {
+        method: "GET",
+      };
+      fetch("/api/eggs", req)
+        .then((res) => {
+          if (!res.ok) throw new Error("something went wrong fetching eggs");
+          return res.json();
+        })
+        .then((res) => {
+          const eggs = res.map((egg) => ({
+            id: +egg.Id,
+            longitude: +egg.longitude,
+            latitude: +egg.latitude,
+          }));
```

Comment:
I noticed that your client has to keep changing these to numbers. You should update your server to do that instead so that your client doesn't have to mess with the data it requested